### DISCUSSION TITLE
Visual scripting editor

### DISCUSSION THREAD

**Discussion by yvanblanchard** [sgenoud/replicad#84] [Visualiser]
Hello,

very nice application, thank you for sharing!
Do you think it would be possible to add a visual scripting (nodes) capability ? At least for basic nodes/features, in order to show how to proceed to complete.

Thank you

**Comment by sgenoud** [comment]
There are no plans as of now to add nodes in the studio, but there were people in the community (@BarbourSmith) working on something in that spirit (but I don't know if a lot of progress was done).

I also have played a bit with [nattod.dev](https://natto.dev/@sgenoud/e2ade00d1fb4406abe78061e7f9a959b), with something a bit closer to visual programming.

P.S. I moved this in a discussion where it feels more appropriate!

**Reply by BarbourSmith** [reply]
This is absolutely something that I am working on and will be building. I had some other things come up that distracted me for a little bit, but I will be making this in the not too distant future.

**Comment by yvanblanchard** [comment]
There is also this lib : https://github.com/jerosoler/Drawflow
